client/invoice_table.js

```diff
@@ -1,17 +1,39 @@
+Template.InvoiceTable.onCreated(function () {
+  var instance = this;
+  limit = new ReactiveVar(20);
+
+  instance.autorun(function () {
+    instance.cursor = instance.subscribe('invoices', date.get(), days.get(), limit.get());
+  });
+});
+
 Template.InvoiceTable.helpers({
   invoices: function () {
-    return InvoiceTicketsCollection.find(filter.get(), {
+    return InvoiceTicketsCollection.find(getFilter(date.get(), days.get()), {
       sort: {
         createdAt: sortState.get('sortCreatedAt', true),
         total: sortState.get('sortTotal', true)
-      }
+      },
+      limit: limit.get()
     });
   },
   createdAtButtonText: function () {
     return sortState.get('sortCreatedAt');
   },
   totalButtonText: function () {
     return sortState.get('sortTotal');
+  },
+  dummy: function () {
+    return "Test";
+  },
+  hasMoreContent: function () {
+    return !(InvoiceTicketsCollection.find(getFilter(date.get(), days.get()), {
+      sort: {
+        createdAt: sortState.get('sortCreatedAt', true),
+        total: sortState.get('sortTotal', true)
+      },
+      limit: limit.get()
+    }).count() < limit.get());
```

Comment:
your `hasMoreContent` helper does not really work, you actually need to get the total count of invoices for the current query from the server! The client can only know how many documents are in the client-cache but not the total available! Think about having millions of invoices, you cannot push all of them to the client just to count them :wink: that would defeat the original purpose of the whole infinite scrolling solution